packages/dds/ordered-collection/src/consensusOrderedCollection.ts

```diff
@@ -185,6 +185,13 @@ export class ConsensusOrderedCollection<T = any>
 		return true;
 	}
 
+	/**
+	 * Return the size of the collection
+	 */
+	public size?(): number {
```

Comment:
Probably prefer "length", not exactly sure why we're calling it "size" internally but "length" seems more traditional here.  Would also recommend doing as a getter rather than standard method, and no need for nullability.

```suggestion
	public get length(): number {
```